### PR TITLE
Update to Minio NAS Gateway

### PR DESCRIPTION
**Pull request by jamesneimanconsulting**
Initial edit

- [X] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

File: docs/gateway/nas.md [hunk 1]
```diff
@@ -1,43 +1,65 @@
 # Minio NAS Gateway [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
-Minio Gateway adds Amazon S3 compatibility to NAS storage. You may run multiple minio instances on the same shared NAS volume as a distributed object gateway.
 
-## Run Minio Gateway for NAS Storage
-### Using Docker
-```
+This topic describes how to use Minio Gateway to add Amazon S3 compatibility to NAS storage.
```
Comment (eco-minio) [inline docs/gateway/nas.md:3]: Change topic to guide

File: docs/gateway/nas.md [hunk 2]
```diff
@@ -1,43 +1,65 @@
...
+
+```sh
 mc config host add mynas http://gateway-ip:9000 access_key secret_key
 ```
 
-### List buckets on nas
-```
+### 3.2 List the Buckets on NAS
+
+Use the following command to list the containers on NAS:
```
Comment (poornas) [inline docs/gateway/nas.md:53]: list the containers -> list buckets

File: docs/gateway/nas.md (hunk 1 above)
Comment (poornas) [inline docs/gateway/nas.md:3]: topic -> guide would be better

**Review by poornas — CHANGES_REQUESTED** [review]

File: docs/gateway/nas.md (hunk 2 above)
Comment (eco-minio) [inline docs/gateway/nas.md:53]: Should change containers to buckets

**Review by eco-minio — CHANGES_REQUESTED** [review]

**Pull request merged**